src/app/board/document/tier/[postTier]/page.tsx

```
@@ -1,54 +1,254 @@
+        pageSize: 10,
+        sortType: filtering.sortType || "LATEST",
+        documentTypes: filtering.documentTypes,
+        postTier: postTier,
+      });
+
+      if (response && response.documentPostsPage) {
+        setDocumentData(response.documentPostsPage.content || []);
+        setTotalPages(response.documentPostsPage.totalPages || 0);
+      }
+    } catch (error) {
+      console.error(`${TIER_DISPLAY_NAME[postTier.toUpperCase()]} 자료를 불러오는데 실패했습니다:`, error);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)